src/pricecypher/rest.py

```diff
@@ -0,0 +1,257 @@
+import json
+import requests
+from time import sleep
+from random import randint
+from datetime import datetime
+from marshmallow import Schema
+
+from .exceptions import PriceCypherError, RateLimitError
+
+UNKNOWN_ERROR = 'pricecypher.sdk.internal.unknown'
```

Comment:
What is this? Will it just return this as a string in the error when no other error message can be extracted? If so, why this string?

Reply:
It's used as a placeholder when a certain part of the error cannot be extracted from the error response. For instance, if the status code cannot be extracted, this is displayed instead of None or whatever. It shouldn't really matter what exact string is used as long as we can identify it, and I think an unknown error of the pricecypher sdk is pretty descriptive since if the error is known, this is not displayed.